### PR TITLE
Allwinner: Sort out SPDIF support

### PR DESCRIPTION
**Pull request by jernejsk**
Enable SPDIF driver in arm64 kernel config, enable SPDIF in Tanix TX6 DT and fix ALSA config which missed one additional SPDIF variant.

Partially addresses forum request: https://forum.libreelec.tv/thread/24802-tanix-tx6-spdif-passthrough

### CODE REVIEW

**Comment by jernejsk** [comment]
Now all user wishes are addressed, but not yet tested. Let's see...

File: projects/Allwinner/filesystem/usr/share/alsa/cards/simple-card.conf [hunk 1]
```diff
@@ -71,6 +71,8 @@ simple-card.pcm.hdmi.0 {
 
 <confdir:pcm/iec958.conf>
 
+simple-card.pcm.iec958."On-board SPDIF" simple-card.pcm.iec958."sun50i-h6-spdif"
```
Comment (knaerzche) [inline projects/Allwinner/filesystem/usr/share/alsa/cards/simple-card.conf:74]: You to prefix the referred card config with the "cards." namespace and the examples have shown it needs to be completly quoted, i.e. "cards.simple-card.pcm.iec958.sun50i-h6-spdif"

Comment (jernejsk) [reply projects/Allwinner/filesystem/usr/share/alsa/cards/simple-card.conf:74]: Fixed.

Comment (jernejsk) [reply projects/Allwinner/filesystem/usr/share/alsa/cards/simple-card.conf:74]: scratch that

Comment (jernejsk) [reply projects/Allwinner/filesystem/usr/share/alsa/cards/simple-card.conf]: now it's fixed

**Review by knaerzche — APPROVED** [review]

**Pull request merged**